courses/free-courses-id.md

```diff
@@ -182,6 +182,7 @@
 * [Belajar Spring Dasar Bahasa Indonesia](https://www.youtube.com/playlist?list=PLiuHSY2x882aeiESAgna5eVa_cOpFnxQm) - Imam Farisi (YouTube)
 * [Spring Framework and Spring Boot Tutorial (Project CRUD)](https://www.youtube.com/playlist?list=PLFfUPa9IV8Lp-Uognr1ALuqlKyxANO77x) - Wafiq Subhi, Mastahcode (Youtube)
 * [SpringBoot](https://www.youtube.com/playlist?list=PLRjWo99hnirwyafPfaxfu0psMR0hUmdQc) - Hendro Steven Tampake, Kelas Koding (YouTube)
+* [TUTORIAL SPRING BOOT - BAHASA INDONESIA](https://www.youtube.com/playlist?list=PL-CtdCApEFH8bV3sla0QfmMnJQ4YX_wR1) - Eko Kurniawan Khannedy, Programmer Zaman Now (YouTube)
```

Comment:
The second video of the playlist (TUTORIAL SPRING CONFIG PROPERTIES (BAHASA INDONESIA) is reserved for paying subscribers: better replace the playlist with the free video (+4 hours) or leave it as it is?

Reply:
Use this link is ok? [https://www.youtube.com/watch?v=VM3rwdMBORY&list=PL-CtdCApEFH8bV3sla0QfmMnJQ4YX_wR1&index=1](https://www.youtube.com/watch?v=VM3rwdMBORY&list=PL-CtdCApEFH8bV3sla0QfmMnJQ4YX_wR1&index=1)

Reply:
> Using this link, is ok? https://www.youtube.com/watch?v=VM3rwdMBORY

Is this resolved?

Not is suitable have videos if haven't +1h and navigable chapters in timeline

Reply:
Resolved.

Video +4h with chapters